config/defines.inc.php

```diff
@@ -24,9 +24,15 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+$debugIps = require __DIR__.DIRECTORY_SEPARATOR.'debug_ip.php';
```

Comment:
If files does not exist, this might throw a warning, isn't it ?

Reply:
Yeah, there is a lot of stuff to be improved, I'm just wondering if this approach for enabling debug based on IP is good and we I can continue it.